src/components/shared/chart/Canvas.js

```diff
@@ -105,16 +105,24 @@ export default class ChartCanvas<HoveredItem> extends React.Component<
     }
   }
 
-  _onMouseMove(event: SyntheticMouseEvent<>) {
+  _hoveredItemFromMouseEvent(event: SyntheticMouseEvent<>): HoveredItem | null {
     if (!this._canvas) {
-      return;
+      throw new Error('Canvas ref not set');
     }
 
     const rect = this._canvas.getBoundingClientRect();
     const x: CssPixels = event.pageX - rect.left;
     const y: CssPixels = event.pageY - rect.top;
```

Comment:
(not for this patch) I wonder if `event.offsetX/Y` would yield the same value, and would prevent the need to use `getBoundingClientRect`.

Reply:
At least when I tried I always got equal values, so it seems right. :-)